index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/skeleton/2.0.4/skeleton.min.css" />
+  <link rel="stylesheet" href="./assets/css/style.css" />
+  <title>COVID-19 Risk Tracker</title>
+</head>
+
+<body>
+  <!-- HEADER SECTION -->
+  <header>
+    <div>
+      <h1>COVID-19 Risk Tracker</h1>
+      <h2>A helpful resource that lets you stay safe and informed when traveling to European countries</h2>
+      <div>
+      <!-- (Where dynamically generated TOTAL infection & vaccination stats will be displayed)-->
+      </div>
+      <div id="jumbotron-image-container">
+        <img>
+      </div>
+    </div>
+  </header>
+
+
+  <!-- MAIN SECTION -->
+  <main>
+    <div class="row">
+
+      <!-- Search History Side Bar -->
+      <section id="search-history-container" class="two columns">
+        <h3><u>Search History</u></h3>
+        <div id="search-history-buttons">
+          <!-- (Where dynamically generated search buttons will be displayed)-->
+        </div>
+      </section>
+
+      <!-- Current Search Section -->
+      <section id="current-search-container" class="ten columns">
+        <h3>Select from the dropdown menu to display <strong>current statistics</strong> and the <strong>risk assessment rating
+        </strong> for a particular European country:</h3>
+        <input />
+        <button id="search-button"><stong>+</stong></button>
+        <div id="current-search-info">
+          <!-- (Where dynamically generated country cards will be displayed)-->
+        </div>
+      </section>
+    </div>
+  </main>
+
+
+  <!-- FOOTER SECTION -->
+  <footer>
+    <h4>For additonal information concerning travel and COVID-19, please visit the <a href="https://www.cdc.gov/coronavirus/2019-ncov/travelers/index.html" target="_blank">CDC Travel Guidelines site.</a></h4>
+  </footer>
+
+
+
+
+<!-- 
```

Comment:
This is incomplete, so the js file won't be useable
"<!-- 
<js> -->"

Reply:
fixed!